src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1225,46 +1265,69 @@ namespace Google.Example.Library.V1.Samples
 
 // Generated code. DO NOT EDIT!
 
-// This is a generated sample ("FlattenedAsync", "test_on_success_map")
+// This is a generated sample ("LongRunningFlattenedAsyncPollUntilComplete", "wap")
 
 // sample-metadata
-//   title: test maps in response handling
-//   description: When there is a very long description for a sample file we can split
-//     the description into multiple lines and the samples will be just fine
-//   usage: dotnet run
+//   title: GetBigBook: 'War and Peace'
+//   description: Testing calling forms
+//   usage: dotnet run [--shelf "Novel\\\"\`\b\t\n\r"]
 
-// [START sample]
+using CommandLine;
+
+// [START hopper]
```

Comment:
This and the corresponding END tag do not enclose a valid syntactic block of code (in the sense that we open some braces below but close them after the the end tag.  How about putting the START/END tags around the sample function?

Reply:
:)